util.go

```diff
@@ -329,3 +329,76 @@ func FullBaseFsPath(basePathFs *BasePathFs, relativePath string) string {
 
 	return combinedPath
 }
+
+func CopyFile(srcFs Fs, srcFilePath string, destFs Fs, destFilePath string) error {
+	// Some code from https://www.socketloop.com/tutorials/golang-copy-directory-including-sub-directories-files
+	srcFile, err := srcFs.Open(srcFilePath)
+	if err != nil {
+		return err
+	}
+	defer srcFile.Close()
+
+	srcInfo, err := srcFile.Stat()
+	if err != nil {
+		return err
+	}
+
+	destFile, err := destFs.Create(destFilePath)
+	if err != nil {
+		return err
+	}
+	defer destFile.Close()
+
+	_, err = io.Copy(destFile, srcFile)
+	if err != nil {
+		return err
+	}
+
+	if err != nil {
```

Comment:
this err has been handled in line 353. 

maybe below is better?
```suggestion
if err := destFs.Chmod(destDirPath, srcInfo.Mode()); err != nil {
        return err
}
```